pallets/abc-demo/src/delegate.rs

```diff
@@ -40,7 +51,7 @@ pub fn request_single_delegate(account: AccountId32) {
             let content = base64::encode(buf);
             let request_url = format!(
                 "{}{}?content={}",
-                SERVICE_BASE_URL, APPLY_DELEGATE, &content
+                get_url(net_address), APPLY_DELEGATE, &content
```

Comment:
If parse net address error, we should abort this request

Reply:
fixed